arch/arm/boot/dts/overlays/merus-amp-overlay.dts

```diff
@@ -57,4 +57,7 @@
 			status = "okay";
 		};
 	};
+		__overrides__ {
```

Comment:
I'm not sure where the extra whitespace came from on this line, but it would be nicer without it.

Reply:
Good spot - think it is because I used the online editor, stupidly. Always seems to mess up formatting.

Reply:
Should be fixed in latest push